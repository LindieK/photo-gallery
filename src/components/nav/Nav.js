import React from "react";

import Search from "../search/Search";

const Nav = (props) => {
  const { searchTerm, /*handleSearchTermChange,*/ handleFormSubmit } = props;
  return (
    <nav>
      <h1>PG</h1>
      <Search
        searchTerm={searchTerm}
        //handleSearchTermChange={handleSearchTermChange}
        handleFormSubmit={handleFormSubmit}
      />
    </nav>
  );
};
export default Nav;
