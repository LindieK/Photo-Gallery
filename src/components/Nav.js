import React from 'react'
import Search from './Search'
import Categories from './Categories'
import '../styles/nav.css'

export default function Nav() {
    return (
        <header>
            <div className='Nav'>
                <h1>PHOTO GALLERY</h1>
                <Search />
            </div>
            <Categories />
        </header>
    )
}
