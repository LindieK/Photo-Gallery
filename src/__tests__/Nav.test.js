import React from 'react';
import { shallow } from 'enzyme';
import Nav from '../components/Nav'

describe('Nav Component', () => {
    it('should render correctly', () => {
        const nav = shallow(<Nav />);

        expect(nav).toMatchSnapshot();
    });
});
