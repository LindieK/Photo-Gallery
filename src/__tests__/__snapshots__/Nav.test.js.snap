// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component should render correctly 1`] = `ShallowWrapper {}`;
